posix.mak

```diff
@@ -484,6 +504,13 @@ ${DOC_OUTPUT_DIR}/library-prerelease/.htaccess : dpl_prerelease_htaccess
 	@mkdir -p $(dir $@)
 	cp $< $@
 
+DMD_EXCLUDE =
+ifeq (osx,$(OS))
+	DMD_EXCLUDE += -e /scanelf/d -e /libelf/d
+else
+	DMD_EXCLUDE += -e /scanmach/d -e /libmach/d
+endif
```

Comment:
DDox tries to generate the docs for all `.d` files. However for dmd this is tricky, because the `{mach, elf, mscoff}` are platform dependent. Thus the need to exclude these files (and similarly for the `objc.d` files).